tests/tests_suite_test.go

```diff
@@ -319,6 +332,13 @@ func (s *existingSspStrategy) GetVmConsoleProxyNamespace() string {
 	return namespace
 }
 
+func (s *existingSspStrategy) GetTektonPipelinesNamespace() string {
+	if s.ssp == nil || s.ssp.Spec.TektonPipelines == nil {
+		panic("Strategy is not initialized")
+	}
+	return s.ssp.Spec.TektonPipelines.Namespace
```

Comment:
So we'll check to `.ssp.Spec.TektonPipelines.Namespace` as well, and return some default value instead of Panic?

Reply:
That depends on Karel's answer to: https://github.com/kubevirt/ssp-operator/pull/532#discussion_r1190924235

If we support empty `.ssp.Spec.TektonPipelines.Namespace`, then there should be the same default as in production code. And if we don't support `""`, then it can panic.